vendor/bat-native-ledger/include/bat/ledger/ledger_client.h

```diff
@@ -78,13 +78,15 @@ class LEDGER_EXPORT LedgerClient {
 
   virtual void LoadPublisherState(OnLoadCallback callback) = 0;
 
-  virtual void OnPanelPublisherInfo(Result result,
-                                   ledger::PublisherInfoPtr publisher_info,
-                                   uint64_t windowId) = 0;
+  virtual void OnPanelPublisherInfo(
+      Result result,
```

Comment:
Maybe add a `ledger` namespace to this for consistency with other parameters.

Reply:
added